docs/pages/billing/manage.mdx

```diff
@@ -62,13 +62,13 @@ To upgrade to a different plan:
 
 - On **Checkout**, you are asked to enter your email, card details, and billing address. After adding these details, click **Pay Now** to subscribe to the new plan.
 
-> **Note**: If you are subscribing to the [On-demand plan](/billing/plans/#on-demand), you are not charged any fee when you check out. Since this is a usage-based plan, all charges are due at the end of the billing period.
+> **Note**: The [Starter plan](/billing/plans/#starter) costs $19 per month and includes $30 of build credit.
 
 ### Downgrade a plan
```

Comment:
Can we also update the image below in this section since it refers to the On-demand plan?

![CleanShot 2025-06-18 at 17 09 26@2x](https://github.com/user-attachments/assets/4be412fb-9a36-411e-bb0a-9bc772821e73)